packages/protocol/contracts/governance/DowntimeSlasher.sol

```diff
@@ -83,7 +83,9 @@ contract DowntimeSlasher is SlasherUtil {
       uint256 signerIndex = epochNumberOfBlock(n, sz) == startEpoch
         ? startSignerIndex
         : endSignerIndex;
-      if (uint256(getParentSealBitmap(n)) & (1 << signerIndex) != 0) return false;
+      // We want to check signers for block n,
+      // so we get the parent seal bitmap for the next block
+      if (uint256(getParentSealBitmap(n + 1)) & (1 << signerIndex) != 0) return false;
```

Comment:
Do we need to change `    require(endBlock < block.number - 1, "end block must be smaller than current block");`?

Reply:
No, I think it's correct now.